### DISCUSSION TITLE
Setup Teams/Slack channels

### DISCUSSION THREAD

**Discussion by jimmarino** [eclipse-edc/.github#15] [Planning]
People are using the Issues list to ask for more detail on the project roadmap and implementation questions.

I propose we do three things:

1. Move design and roadmap discussions to a specific chat channel (Teams or Slack). We could repurpose the "General" Teams channel for this.

2. Create a separate channel for people to post questions.

3. Only use issues to file bug requests, feature requests, etc.

@mspiekermann, I assigned this to you so we can discuss it in the next committers meeting.

**Comment by MoritzKeppler** [comment]
what about https://docs.github.com/en/discussions ?

**Comment by mspiekermann** [comment]
It has advantages for the community to have the information in one single location.  Github discussions are perfect for this, that's why we just activated the Discussions-Tab (https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/discussions).  

Ofc we are happy to receive suggestions for more categories etc.
